libnavui-voice/src/main/java/com/mapbox/navigation/ui/voice/utils/Utils.kt

```diff
@@ -0,0 +1,16 @@
+package com.mapbox.navigation.ui.voice.utils
+
+import com.mapbox.api.directions.v5.models.VoiceInstructions
+import com.mapbox.navigation.ui.voice.api.VoiceInstructionsParser
+
+internal object Utils {
+
+    private val firstInstructionsStarters = setOf("head ", "drive ", "walk ", "bike ")
+
+    fun isFirstVoiceInstruction(voiceInstructions: VoiceInstructions): Boolean {
```

Comment:
I don't think this is valid because of different languages. Moving this to draft while I discuss the first instruction detection with the NAVAPI team.